common/static/common/js/spec/components/list_spec.js

```diff
@@ -0,0 +1,67 @@
+define(['jquery', 'backbone', 'underscore', 'common/js/components/views/list'],
+    function ($, Backbone, _, ListView) {
+        'use strict';
+        describe('ListView', function () {
+            var Model = Backbone.Model.extend({
```

Comment:
I've found that as our specs grow, we often have variable assignments quite separated from their declarations. I find this form easier for locating a variable assignment. At least, in pycharm, when you 'go to definition' it will take you to the actual assignment instead of the declaration.